app/src/main/java/org/p2p/wallet/bridge/send/ui/dialog/BridgeSendFeeBottomSheet.kt

```diff
@@ -84,15 +86,49 @@ class BridgeSendFeeBottomSheet : BaseDoneBottomSheet() {
         textViewFiatAmount.isVisible = false
     }
 
+    private fun ItemClaimDetailsPartBinding.bindWillGetAmountLine(title: String, fee: BridgeAmount) {
+        textViewTitle.text = title
+        textViewFiatAmount.text = fee.fiatAmount.orZero().asUsd()
+        bindFreeOrValue("${fee.tokenAmount.orZero().formatToken(fee.tokenDecimals)} ${fee.tokenSymbol}")
+    }
+
     private fun ItemClaimDetailsPartBinding.bindDetailsLineWithFee(title: String, fee: BridgeAmount) {
         textViewTitle.text = title
         textViewFiatAmount.text = fee.fiatAmount?.asUsd() ?: getString(R.string.bridge_info_transaction_free)
-        val formattedTokenAmount = fee.formattedTokenAmount
-        if (formattedTokenAmount == null) {
+        bindFreeOrValue(fee.formattedTokenAmount)
+    }
+
+    private fun ItemClaimDetailsPartBinding.bindDetailsTotalFee(title: String, fees: List<BridgeAmount>) {
```

Comment:
looks like logic that should be in presenter
view should be stupid as possible

Reply:
hmmm you are right will move it